File: java/BillsApi/src/main/java/com/apilangarena/javabillsapi/application/bills/get/BillSummaryDto.java
package com.apilangarena.javabillsapi.application.bills.get;

import java.math.BigDecimal;
import java.time.LocalDate;

public record BillSummaryDto(
    Long id,
    String billNumber,
    LocalDate issuedAt,
    BigDecimal total,
    String currency
) {
}

File: kotlin/BillsApi/src/main/kotlin/com/apilangarena/billsapi/application/bills/get/BillSummaryDto.kt
package com.apilangarena.billsapi.application.bills.get

import java.math.BigDecimal
import java.time.LocalDate

data class BillSummaryDto(
    val id: Long,
    val billNumber: String,
    val issuedAt: LocalDate,
    val total: BigDecimal,
    val currency: String,
)
